tokio/src/loom/std/atomic_u64_as_mutex.rs

```diff
@@ -1,18 +1,24 @@
 use crate::loom::sync::Mutex;
 use std::sync::atomic::Ordering;
 
+cfg_has_const_mutex_new! {
+    #[path = "atomic_u64_static_const_new.rs"]
```

Comment:
I'm experimenting with a way to have less code **within** the `cfg` macros. This should help `rustfmt` apply to more code.